### PR TITLE
XRT-464 enhance XRT with verbose logs

### PR DESCRIPTION
**Pull request by xdavidz**
print logs for every 1M data transferred, it makes our live easier for debugging.

### CODE REVIEW

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/ospi_versal.c [hunk 1]
```diff
@@ -64,6 +64,8 @@
 #include "../xocl_drv.h"
 #include "xrt_drv.h"
 
+// #define OSPI_VERBOSE
```
Comment (larry9523) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/ospi_versal.c:67]: I think it is better not to use "c99" comments style in kernel code

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/ospi_versal.c [hunk 2]
```diff
@@ -84,6 +86,13 @@ struct ospi_versal {
 #define	OV_INFO(ov, fmt, arg...)    \
 	xocl_info(&ov->ov_pdev->dev, fmt "\n", ##arg)
 
+#ifdef OSPI_VERBOSE
```
Comment (larry9523) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/ospi_versal.c:89]: any concern we need to use #ifdef here? Though without this, we can turn the debug information without recompiling the code.

**Comment by larry9523** [comment]
Also, you may want to update the year of copyright

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/ospi_versal.c (hunk 2 above)
Comment (xdavidz) [reply src/runtime_src/core/pcie/driver/linux/xocl/subdev/ospi_versal.c]: Agree, so just remove the ifdef for deb logs.

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/ospi_versal.c (hunk 1 above)
Comment (xdavidz) [reply src/runtime_src/core/pcie/driver/linux/xocl/subdev/ospi_versal.c]: removed

**Comment by xdavidz** [comment]
> Also, you may want to update the year of copyright

Fixed the copyright year. Also removed unnecessary ifdef.

**Pull request merged**